apps/src/blockly/customBlocks/googleBlockly/mutators/behaviorDefMutator.js

```diff
@@ -0,0 +1,206 @@
+// import {CdoParameterModel} from './parameterModel';
```

Comment:
would be good to add comments to this file and `behaviorGetMutator` explaining what they do